hydroflow_plus_test/examples/perf_compute_pi.rs

```diff
@@ -50,10 +49,11 @@ async fn main() {
         &DeployProcessSpec::new(|| {
             let mut deployment = deployment.borrow_mut();
             let host = create_host(&mut deployment);
-            let perf_options = PerfOptions {
-                output_file: PathBuf::from("leader.perf"),
-                frequency: 5,
-            };
+            let perf_options = PerfOptions::builder()
+                .fold_outfile("leader.data.folded")
```

Comment:
Will this still work on Localhost? Or will it error out because a perf outfile was not given.

Reply:
Ah good catch, that will error, although it will always error for me since I'm on windows since `perf` doesn't exist, but will fix that next